python/pyarrow/_flight.pyx

```diff
@@ -950,7 +972,9 @@ cdef class FlightInfo(_Weakrefable):
                 f"descriptor={self.descriptor} "
                 f"endpoints={self.endpoints} "
                 f"total_records={self.total_records} "
-                f"total_bytes={self.total_bytes}>")
+                f"total_bytes={self.total_bytes} "
+                f"ordered={'true' if self.ordered else 'false'} "
```

Comment:
Nitpicking, but I'd expect to have `True` and `False` in a Python repr, as the builtin bool constants are capitalized.

Reply:
Makes sense, I'll flag this with the arrow devs.